### DISCUSSION TITLE
Using Zoraxy with Certify

### DISCUSSION THREAD

**Discussion by ChrisCraddock** [tobychui/zoraxy#186] [Q&A] [answered]
Good morning,

According to a post as well as the Certbot Github, Windows is no longer supported.  Are their any plans to be able to link/connect the bottom ACME portion of Zoraxy > TLS/SSL Open ACME Tool with Certify?

**Comment by tobychui** [comment]
No because I don't use Certify. You can't use Lets Encrypt for some reason or why you are asking?

**Reply by ChrisCraddock** [reply, marked as answer]
For some reason it was pulling up a gray screen when I tried to access it.  And for why I am asking? This is the Q&A section so don't questions go here?  Unless you create a discord instead which would probably be helpful.  The ACME screen pulls up fine when swapping to a different browser so I assume it was something to do with where I was accessing it from. Everything (well on that end) is fine and can be closed.

**Reply by tobychui** [reply]
Yes, this is the place to ask questions. I am just interested to know why you can't use Let's Encrypt (which Certify is also based on them). 
Anyway, if this is solved then I am closing this now.

**Reply by ChrisCraddock** [reply]
Apologies for the late reply, it turns out it must have been the browser I was on where it did not show but once I got home from work it showed up.  I am in the process of setting up a new lab and excited to incorporate Zoraxy in it!
